docs/ballerina-by-example/examples/json-struct-map-conversion/json-struct-map-conversion.sh

```diff
@@ -1,3 +1,5 @@
+#At the command line, navigate to the directory that contains the 
+#`.bal` file and run the `ballerina run` command.
```

Comment:
can we replace ` character with ' or " please? ` character has a special meaning in scripting. Users might interpret the words `foo` differently than 'foo' or "foo"

Reply:
Added that character to make the word appear in grey-monospaced format on the html page. Generally we use monospace format in docs for file paths, file names, parameter names etc. 

So, is that okay?